src/main/java/name/valery1707/kaitai/KaitaiMojo.java

```diff
@@ -1,27 +1,24 @@
 package name.valery1707.kaitai;
 
-import org.apache.commons.io.FilenameUtils;
-import org.apache.commons.lang3.SystemUtils;
 import org.apache.maven.execution.MavenSession;
 import org.apache.maven.plugin.AbstractMojo;
 import org.apache.maven.plugin.MojoExecutionException;
-import org.apache.maven.plugin.MojoFailureException;
-import org.apache.maven.plugin.logging.Log;
 import org.apache.maven.plugins.annotations.LifecyclePhase;
 import org.apache.maven.plugins.annotations.Mojo;
 import org.apache.maven.plugins.annotations.Parameter;
 import org.apache.maven.project.MavenProject;
 import org.apache.maven.settings.Settings;
+import org.slf4j.Logger;
+import org.slf4j.impl.StaticLoggerBinder;
 
 import java.io.File;
-import java.net.MalformedURLException;
 import java.net.URL;
-import java.nio.file.Files;
 import java.nio.file.Path;
-import java.util.*;
+import java.util.Arrays;
+import java.util.List;
 
 import static java.lang.String.format;
-import static name.valery1707.kaitai.MojoUtils.*;
+import static name.valery1707.kaitai.KaitaiUtils.*;
```

Comment:
\[Checkstyle\] WARNING: Using the '\.\*' form of import should be avoided \- name\.valery1707\.kaitai\.KaitaiUtils\.\*\.